conductor/src/test/java/com/bluelinelabs/conductor/util/AttachFakingFrameLayout.java

```diff
@@ -126,4 +130,34 @@ public void onViewRemoved(View child) {
         super.onViewRemoved(child);
     }
 
+    @Override
+    public boolean post(Runnable action) {
+        if (needDelayPost) {
+            delayedPosts.add(action);
+        } else {
+            return super.post(action);
+        }
+
+        return true;
+    }
+
+    public void runDelayedPosts() {
+        for (Runnable runnable : delayedPosts) {
+            runnable.run();
```

Comment:
Why not enqueue to `super.post(runnable)` here instead of `runnable.run()`?

Reply:
What do you mean?

Reply:
Please, show some code example

Reply:
in `post` method you have this code:
```java
if (needDelayPost) {
            delayedPosts.add(action);
        } else {
            return super.post(action);
        }
```
why not just use View's method `post` to execute the runnables here instead of manually triggering them in the for loop?

Reply:
I still not understand the full picture